.circleci/config.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       - run:
           name: Lint Go
           command: |
-            go get -u golang.org/x/lint/golint
+            go get golang.org/x/lint/golint@v0.0.0-20201208152925-83fdc39ff7b5
```

Comment:
Pinned to previous `golint` version since the latest version would fail with:
```
/go/pkg/mod/golang.org/x/tools@v0.8.0/go/gcexportdata/gcexportdata.go:97:9: undefined: io.ReadAll
note: module requires Go 1.18
```
A proper fix would be to upgrade to the latest Go.

Reply:
Yes please, open an issue to bump go :-).